api/agent_test.go

```diff
@@ -77,7 +77,7 @@ func TestAgent_Join(t *testing.T) {
 	})
 	defer s2.Stop()
 
-	// Attempting to join a non-existent host returns error
+	// Attempting to join a non-Existent host returns error
```

Comment:
Eww, that probably means I failed to replace some instances of `Existant` with `Existent`, sorry :-(

While it might seem like this is robotic, it's done by me manually doing things, and I tend to cheat when dealing w/ pairs of `existant (existant, Existant)`.

Sorry :-(